### DISCUSSION TITLE
responsive columns on mobile

### DISCUSSION THREAD

**Discussion by bruce-gemberling** [breezefront/community#28] [1. Help]
On my homepage on desktop I have a row with three columns which all have a width of 33%. When I view the same row/columns on mobile the width stays at 33% instead of switching to 100% width and stacking the column items. I know how to fix this in bootstrap but am unsure how to fix this in this theme. 

Help!?!?!!

**Comment by vovayatsyuk** [comment]
This issue is caused by a breaking change in the latest magento’s pagebuilder (2.4.5). I’ve already fixed it in the repo ([1](https://github.com/breezefront/theme-frontend-breeze-blank/commit/2e44a9778f59d98bdfca656e4c5060a30324804a), [2](https://github.com/breezefront/theme-frontend-breeze-blank/commit/0a0fde80e6449d6612f64d46b81cf44cfa8bc531)) and going to publish breeze 2.0 in a few days.
